src/pages/NotebooksPage/index.js

```diff
@@ -16,6 +17,98 @@ import data from '../../data/database';
 
 class NotebooksPage extends React.Component {
 
+  state = {
+    title: '',
+    disable: true,
+    loading: false,
+    open: false,
+    visible: false
+  };
+
+  /**
+   * This method handles what happens when the name value changes in the input field.
+   *
+   * @param {Event} event
+   *
+   * @returns {void}
+   */
+  onInputChange = (event) => {
+    let value = event.target.value;
+
+    this.setState({
```

Comment:
```suggestion
    this.setState({title: value})
```
setState will handle merging of state for you